### PR TITLE
[server] WIP : reactivate flaky server tests for WMS requests

### PR DESCRIPTION
**Pull request by pblottiere**
## Description

Some WMS tests have been deactivated in Travis because of their flaky behaviors. The aim of this PR is to detect underlying issues.

For now, I just splitted some tests to have a better vision of what is happening in Travis.

## Checklist

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and containt sufficient information in the commit message to be documented
- [ ] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] New unit tests have been added for core changes
- [ ] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

**Review by rldhont — APPROVED** [review]
Can be merged

**Comment by pblottiere** [comment]
I think that @m-kuhn disabled these tests in the first place so I would like its opinion before merging this PR. 

To be clear, I didn't fix anything!

I just splitted (and 'unskipped') a very big test (on `GetProjectSettings`, `GetFeatureInfo`, `GetStyles`, `GetCapabilities`, and so on) in order to detect issues more easily. Moreover, I reactivated a test on `GetPrint` since this request has been recently refactored (so perhaps the issue is gone?). 

FWIW, I ran Travis several times without any errors (I'm still waiting the result for the last commit).

In short, flaky tests are still there but I hope that this PR will help us to detect the cuplrit test :).

Matthias, let me know what you think about it.

**Comment by rldhont** [comment]
I have tested and passed all the tests locally with #5405 so do you want I merge it and then #5399
has to be updated #5400 or the oposite ?

**Comment by rldhont** [comment]
I have merged so you can rebased this PR

**Comment by m-kuhn** [comment]
I very much like it, thanks a lot @pblottiere . I hate disabling tests, I see it as a necessary evil because failing/flaky tests affect the whole committer community. So bit +1 for taking responsibility here!
